core/src/main/java/google/registry/model/domain/DomainBase.java

```diff
@@ -137,6 +137,7 @@
    *
    * @invariant fullyQualifiedDomainName == fullyQualifiedDomainName.toLowerCase(Locale.ENGLISH)
    */
+  @Column(name = "domainName")
   @Index String fullyQualifiedDomainName;
```

Comment:
Add a TODO here to rename this once the Cloud SQL migration is complete.

Reply:
Done (also in HostBase)